generators/client/files-common.js

```diff
@@ -62,7 +62,7 @@ const commonFiles = {
       ],
     },
     {
-      condition: generator => generator.enableI18nRTL,
+      condition: generator => generator.enableI18nRTL && !generator.clientFrameworkReact,
```

Comment:
Why do we skip this? Is it all handled by the post-rtlcss plugin now?

Reply:
Yes.
bootstrap 5 have native rtl support using rtlcss.
All in one css is not officially supported, but looks to work great.